### ISSUE TITLE
Give an example of unit test

### ISSUE THREAD

**Issue by ripzery** [TeamTechnologies/security-workshop-sample#42]
It would be nice if it has a test..

**Comment by yakivmospan** [comment]
Yes, probably would be nice to make some for Encrypting/Decrypting, but this consumes time, if I got some I will add them. Meanwhile, if you have time and a desire, contribution is welcomed ;)!
